docs/index.html

```diff
@@ -137,8 +137,20 @@
 
             
         </li>
-    
-        <li class="chapter " data-level="1.4" >
+        <li class="chapter " data-level="1.3" data-path="faq.html">
```

Comment:
This file is not used in production - please remove this change.

Reply:
fixed :)